README.md

```diff
@@ -1,22 +1,30 @@
 ----
-
-Applications running on the POD on Azure Container Service (AKS/ACS Engine) require access to identities in Azure Active Directory (AAD) to access resources that use an identity provider. AAD provides a construct called a Service Principal that allows applications to assume identities with limited permissions, and Managed Service Identity (MSI) - automatically generated and rotated credentials that easily retrieved by an application at run-time to authenticate as a service principal. 
-A cluster admin configures the Azure Identity Binding to the Pod. Without any change of auth code the application running on the pod works on the cluster.
+Aad Pod Identity enables applications running in pods on AKS or aks-engine Kubernetes clusters to securely access cloud resources, by leveraging Azure Active Directory(AAD). Administrators can configure identities and bindings, to match pods with those identities. Following this, without any code modifications, the applications running within the pod can access the appropriate cloud resources. The administrator interactions with the aad-pod-identity are via Kubernetes primitives.
```

Comment:
can we replace `AKS or aks-engine Kubernetes clusters` with `Kubernetes clusters running on Azure (managed or unmanaged)`? The idea is, it can be used with kubernetes + azure

Can we replace ` access the appropriate cloud resources` to ` access the any application that depend on Azure Active Directory as an Identity Provider`